README.md

```diff
@@ -22,6 +22,7 @@ The following client/verifier versions should be used with the asset classificat
 
 | Client / Verifier | AC Smart Contract |
 |-------------------|-------------------|
+| v3.1.0+           | v3.1.0+           |
```

Comment:
The old v3.0.0 client is compatible with new contract versions, but cannot use the new `addOsGatewayPermission` flag, so these values are bit confusing but make sense in that context